2-mandatory/6-journey-planner.js

```diff
@@ -81,7 +85,9 @@ function getTransportModes() {}
 
   Hint: Use the corresponding array method to decide if an element is included in an array.
 */
-function isAccessibleByTransportMode() {}
+function isAccessibleByTransportMode(arr,str) {
```

Comment:
I think we all need to work on how we name our variables to make it easier for readers